buf.yaml

```diff
@@ -26,6 +26,8 @@ lint:
 breaking:
   use:
     - FILE
+  except:
+    - FIELD_NO_DELETE
```

Comment:
It would be great if we had some comments about this line indicating that this still requires reserving the name and number. Could you add quickly in a separate PR @technicallyty ?